core/arch/arm/kernel/static_ta.c

```diff
@@ -126,6 +126,7 @@ static void static_ta_enter_close_session(struct tee_ta_session *s)
 	if (s->ctx->ref_count == 1)
 		stc->static_ta->destroy_entry_point();
 	tee_ta_pop_current_session();
+	free(stc);
```

Comment:
This free should be done in `static_ta_destroy()` instead to avoid "use after free".